example/basic_example.py

```diff
@@ -104,15 +106,16 @@ def parse_arguments():
     # This call sets up the MeterProvider, with a PeriodicExportingMetricReader that exports every 5000 ms
     # and the Dynatrace exporter exporting to args.endpoint with args.token
     logger.debug("setting up global OpenTelemetry configuration.")
-    _metrics.set_meter_provider(MeterProvider(
+    metrics.set_meter_provider(MeterProvider(
         metric_readers=[PeriodicExportingMetricReader(
```

Comment:
It seems to me that all of the Temporalities are `CUMULATIVE` if set up this way.

Reply:
Yes, this was an oversight. Thought it was still working the way it did before, but it does not.
Added temporality preference in [2d01775](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/2d01775f272252daa002b671d1a760238bbcfc8c).